test/cmd/newapp.sh

```diff
@@ -36,6 +36,8 @@ os::cmd::expect_success_and_text 'oc new-app mysql --dry-run' 'tag "5.7" for "my
 # test deployments are created with the boolean flag and printed in the UI
 os::cmd::expect_success_and_text 'oc new-app mysql --dry-run --as-test' 'This image will be test deployed'
 os::cmd::expect_success_and_text 'oc new-app mysql -o yaml --as-test' 'test: true'
+os::cmd::expect_success_and_text 'oc new-app mysql --as-test' 'Application is not exposed'
```

Comment:
I think that if you add `--dry-run` here, it'll save having to clean anything up in the test

Reply:
Dry run does not work as we terminate all new-app stuff before we reach output phase. And it makes sense as we dont have what to "expose"
```
	if !o.Action.Verbose() || o.Action.DryRun {
		return nil
	}
```

Reply:
Ah - then do you need to add some `os::cmd::expect_success 'oc delete all --selector="label=something"'` lines?

Reply:
Added one more new small template as reusing old ones was causing lot of false/positives. 
+Added cleaning... This file is so sensitive... you touch one test and 5 starts failing :D

Reply:
done.